rust/operator-binary/src/operations/graceful_shutdown.rs

```diff
@@ -0,0 +1,38 @@
+use snafu::{ResultExt, Snafu};
+use stackable_nifi_crd::NifiConfig;
+use stackable_operator::builder::PodBuilder;
+use std::collections::BTreeMap;
```

Comment:
Move these to the top.

Reply:
https://github.com/stackabletech/nifi-operator/pull/528/commits/547eb6cf8a46867185bc999daa7906237f33daa7